requirements-dev.txt

```diff
@@ -13,6 +13,7 @@ types-requests==2.31.0.2
 types-setuptools
 
 # testing
+datasets
```

Comment:
are these changes going to be moving upstream at some point? if not, then it'll be better to tease out this change to requirements into something specific to Neuralmagic.

Reply:
As discussed in our meeting, leaving this here for now.